frontend/src/app/components/list-modal/list-modal.component.ts

```diff
@@ -25,6 +25,7 @@ import {
   AddPostDialog,
 } from 'src/app/components/add-post-modal/add-post.component';
 import User from 'src/app/models/user';
```

Comment:
Duplicate?

Reply:
I'm getting a few duplicate errors (i.e., for Post, PostType, and User) when the building frontend

Reply:
I think those occurred when i merged the develop branch. Fixed now.

Reply:
Hopefully it works properly now.